File: Begginers/C++/salario_1008.cpp
#include <stdio.h>
 
int main() {
 
    int a, b, number;
    double c, salary;
    
    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%lf", &c);
    
    salary = b * c;
    
    printf("NUMBER = %d\n", a);
    printf("SALARY = U$ %.2lf\n", salary);
 
    return 0;
}

File: Begginers/Java/salario_1008.java
import java.io.IOException;
import java.util.Scanner;
import java.util.Locale;

public class Main {
    public static void main(String[] args) throws IOException {
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int number, horas;
        double valor, salario;

        number = sc.nextInt();
        sc.nextLine();
        horas = sc.nextInt();
        sc.nextLine();
        valor = sc.nextDouble();

        salario = valor * horas;

        System.out.printf("NUMBER = %d%n", number);
        System.out.printf("SALARY = U$ %.2f%n", salario);
        
        sc.close();

    }
}
